modules/dataframe/include/inviwo/dataframe/io/csvreader.h

```diff
@@ -89,6 +89,14 @@ class IVW_MODULE_DATAFRAME_API CSVReader : public DataReaderType<DataFrame> {
     CSVReader& setNumberOfExampleRows(size_t rows);
     size_t getNumberOfExamplesRows() const;
 
+    /** @see CSVReader::defaultRowComment */
```

Comment:
this comment doesn't explain what it is doing/what the purpose is. The member only refers back to this function.

Reply:
no it refers to  defaultRowComment not setRowComment.

Reply:
```c++
    /** @see CSVReader::setRowComment */
    static constexpr std::string_view defaultRowComment = "";
```